docker/dev/nginx-dev.conf

```diff
@@ -0,0 +1,71 @@
+worker_processes 1;
```

Comment:
Would just call this file nginx.conf, as we are already in dev folder. Same above for Lemmy config.